.gitignore

```diff
@@ -0,0 +1,43 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Unnecessary file addition, not really relevant to this PR
Sorry

Reply:
Sure, no prob jehjeh, will remove it :+1: